package.json

```diff
@@ -96,7 +96,7 @@
     "rollup-plugin-copy": "^3.3.0",
     "rollup-plugin-sass": "^1.1.0",
     "rollup-plugin-uglify": "^6.0.4",
-    "sass": "^1.33.0",
+    "node-sass": "^4.13.1",
```

Comment:
why this change? Just explain here.

Reply:
It was node-sass befor I tried to fix #2177. I updated to sass because then I would be able to use the @rules. So I'm reverting this also to the state from around 3 weeks.

Reply:
OK, makes sense